docs/blog/posts/2025-02-24-terms-of-service.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Introducing our new Terms of Service
+description: PyPI is formalizing our policies to help us move forward with new services.
+authors:
+  - ewdurbin
+date: 2025-02-25
+tags:
+  - policies
+  - transparency
+---
+
+We're introducing a new
+[Terms of Service](https://policies.python.org/pypi.org/Terms-of-Service/)
+to formalize our relationship to users
+and enable us to move forward with providing new features and services,
+specifically
+[Organization Accounts](https://docs.pypi.org/organization-accounts/).
+
+<!-- more -->
+
+PyPI has had some form of [Terms of Use](https://policies.python.org/pypi.org/Terms-of-Use/)
+document for users since it
+[began accepting uploads in 2005](https://github.com/pypi/legacy/commit/b139c00cfc5794159afb1fc185d77dbc5fc1a2a4#diff-a67499b048e6bb6ef08d44c7a3c541199615b68e3bd153eb0ccedc492e3dec9dR7-R13)
+and has only been updated twice[^1] since.
+These terms have primarily served to protect PyPI
+and the Python Software Foundation (PSF) who operates it.
+
+Over time we have introduced additional policies to protect our users and community
+such as our
+[Code of Conduct](https://policies.python.org/python.org/code-of-conduct/)
+[Privacy Notice](https://policies.python.org/pypi.org/Privacy-Notice/)
+and
+[Acceptable Use Policy](https://policies.python.org/pypi.org/Acceptable-Use-Policy/).
+
+Our new
+[Terms of Service](https://policies.python.org/pypi.org/Terms-of-Service/)
+formalizes our relationship to PyPI users,
+makes protections for the PSF and PyPI users more explicit,
+and establishes terms we need to provide
+[Organization Accounts](https://docs.pypi.org/organization-accounts/)
+to paid
+[Corporate Organizations](https://docs.pypi.org/organization-accounts/pricing-and-payments/#corporate-organizations).
+
+We have worked with our legal team to retain compatibility with the existing
+[Terms of Use](https://policies.python.org/pypi.org/Terms-of-Use/)
+while adding as permissive a set of new terms as possible to ensure that PyPI users 
+and the PSF are protected.
```

Comment:
Is it worth noting here that the ToU are replaced/superseded by the newer ToS? Nowhere is that explicitly stated in the blog yet.